### ISSUE TITLE
Support ASLR properly in trace file

### ISSUE THREAD

**Issue by torusrxxx** [x64dbg/x64dbg#2618]
The trace file should have a list of module info so that the trace file can be used when ASLR is enabled, or even be used in DLL debugging where the user traces the DLL in an application and then read it while the DLL is loaded via x64dbg DLL loader.

**Comment by mrexodia** [comment]
Yeah I think the format can be extended to support ModLoad/ModUnload events (and also exceptions?)
